package.json

```diff
@@ -3,11 +3,6 @@
 	"version": "0.3.2",
 	"module": "index.js",
 	"main": "dist/index.js",
-	"exports": {
```

Comment:
@lukas-schardt I'm not sure if this will break CDN builds.

Reply:
@eddyloewen How can we find out?

Reply:
Nice! 👍